compiler/rustc_middle/src/mir/interpret/value.rs

```diff
@@ -56,15 +56,6 @@ impl<'tcx> ConstValue<'tcx> {
         }
     }
 
-    pub fn try_to_str_slice(&self) -> Option<&'tcx str> {
```

Comment:
@oli-obk: is this useful to have around, or can it be removed?

Reply:
Nope, this can go